### ISSUE TITLE
[Bug] pyenv has updated install instructions for fish

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#423]
As per https://github.com/pyenv/pyenv:

```bash
status is-login; and pyenv init --path | source
status is-interactive; and pyenv init - | source
```

Slight change, but seems to be necessary for the latest fish + pyenv.

Easy fix, just update https://github.com/webinstall/webi-installers/blob/0864a0a0f8ffd54760b5d991314ba00292cc10e1/pyenv/install.sh#L29-L34

**ALSO**: This must be appended to the **BOTTOM** of the config file **NOT** the top.

**Comment by Simba-97** [comment]
Hey @coolaj86 can I work on this issue?

**Comment by coolaj86** [comment]
Yes, please do.

**Comment by coolaj86** [comment]
This was fixed with #425. Just forgot to close the issue.
